README.md

```diff
@@ -34,31 +34,23 @@ git co -b my-feature-branch
 ### 4. Install dependencies
 
 ```
-npm i                         // if new repo
-npm i --global lerna@^5.6.2   // if you didn't do this previously
+npm i                                 // if new repo
+./node_modules/.bin/lerna --version   // Use Lerna locally instead of globally
```

Comment:
I suggest either keeping this line and changing the comment to "verify that Lerna is installed locally" or deleting this line altogether. Thoughts?

Also it can be called out to users of this repo that Lerna doesn't need to be installed globally any more, so it can be deleted.

Reply:
1. change comment to avoid confusion
2. I will add a note for second